doc/03-component-interfaces/03-component-interfaces.md

```diff
@@ -132,7 +132,7 @@ hierarchy diagram using whatever tools you would like. Then, include a picture
 of it in this folder. You may also embed it just below using markdown syntax
 (i.e., `![ALT TEXT](path/to/file)`).
 
-<!-- TODO: make a diagram of your component hierarchy then delete this comment -->
+C:\Users\ns63u\OneDrive\Documents\GitHub\ComponentCreation\doc\03-component-interfaces\HierarchyDiagram.pdf
```

Comment:
I can't see this. You would need to commit it or embed a proper image (JPG or PNG).